packages/components/src/unit-control/utils.ts

```diff
@@ -302,17 +309,17 @@ export const useCustomUnits = ( { units, availableUnits, defaultValues } ) => {
  * accurately displayed in the UI, even if the intention is to hide
  * the availability of that unit.
  *
- * @param {number|string}            currentValue Selected value to parse.
- * @param {string}                   legacyUnit   Legacy unit value, if currentValue needs it appended.
- * @param {Array<WPUnitControlUnit>} units        List of available units.
+ * @param  currentValue Selected value to parse.
+ * @param  legacyUnit   Legacy unit value, if currentValue needs it appended.
+ * @param  units        List of available units.
  *
- * @return {Array<WPUnitControlUnit>} A collection of units containing the unit for the current value.
+ * @return A collection of units containing the unit for the current value.
  */
 export function getUnitsWithCurrentUnit(
-	currentValue,
-	legacyUnit,
-	units = ALL_CSS_UNITS
-) {
+	currentValue: number | string,
```

Comment:
I would probably add a new type in `types.ts`:

```ts
type Value = number | string;
```

And then replace it everywhere we currently intend to represent such a value (including in the `WPUnitControlUnit` and `UseCustomUnitsProps` types).

Reply:
I like that idea, it neatens things a fair bit and ensures consistency (previously I see I'd inconsistently used `number | string` and `string | number`). I've updated, but please let me know if I've missed anything.